src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -26,6 +26,13 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 
 	try {
 		const url = new URL(`${req.protocol}://${req.get("host")}${req.originalUrl}`);
+		if (!githubToken || (!servers.hasCloudServer && !servers.gheServerInfos.length)) {
```

Comment:
A configured customer without a token is valid.

If they don't have a token, they will get kicked off to authenticate... then they will reload the options page and it should catch the unconfigured state then

```suggestion
		if (!servers.hasCloudServer && !servers.gheServerInfos.length) {
```

Reply:
actually, hmmmm that might not work.

Reply:
This route `GitHubCreateBranchOptionsGet` is not inside `GitHubRouter`, so the auth dance doesn't happen at this moment.

Reply:
ok, in this scenario, we need the githubtoken to exist otherwise we aren't authenticated against server or cloud so we dont know if the apps have been configured or not so its full circle

`githubToken && !servers.hasCloudServer && !servers.gheServerInfos.length`

So the variations as I see them
i've installed cloud but not configured(same for single server)
With token: i come here
without token: i login to github, try hit this page again then come here

With multiple servers or cloud and server - none configure
With token: i come here
without token: i login to github, try hit this page again then come here

Reply:
Updated and removed the `gitHubToken` check